src/app/functions/server/getUserBreaches.ts

```diff
@@ -36,6 +43,25 @@ export async function getUserBreaches({
   const breachesData = await getAllEmailsAndBreaches(subscriber, allBreaches);
   appendBreachResolutionChecklist(breachesData, options);
 
+  const ssnBreaches: HibpLikeDbBreach[] = [];
+  const passwordBreaches: HibpLikeDbBreach[] = [];
+  const phoneBreaches: HibpLikeDbBreach[] = [];
+  for (const { breaches } of breachesData.verifiedEmails) {
+    breaches.forEach((b) => {
+      if (b.DataClasses.includes(BreachDataTypes.SSN)) {
+        ssnBreaches.push(b);
+      }
+
+      if (b.DataClasses.includes(BreachDataTypes.Passwords)) {
+        passwordBreaches.push(b);
+      }
+
+      if (b.DataClasses.includes(BreachDataTypes.Phone)) {
+        phoneBreaches.push(b);
+      }
+    });
```

Comment:
I was trying to come up with a more legible/DRY solution, but I think this works! 

My one edge case is if I have a breach that includes both SSN and Passwords, would that specific breach be in both arrays? If so, is that okay? (Question may be for @codemist)

Reply:
The assumption is that it would count as both, thus in both arrays.